exampleVault/Advanced Examples/DnD 5e Ability Scores and Modifiers.md

```diff
@@ -0,0 +1,41 @@
+---
+STR: 8
+DEX: 16
+CON: 11
+INT: 12
+WIS: 14
+CHR: 11
+---
+
+This example calculates a DnD character's ability modifiers from the ability scores and displays it all in a list.
+
+The modifiers are stored in `memory` and not in the frontmatter. This reduces clutter but requires some hidden fields that calculate the values while the fields in the list just format the value to always contain a `+/-` sign.
+
+
+## Character ability scores
+
+```meta-bind
+VIEW[ floor(({STR}-10)/2) ][math(hidden):memory^STR_mod]
```

Comment:
i would do the spacing like this `VIEW[floor(({STR} - 10) / 2)]`, for better readability